indicadores/forms.py

```diff
@@ -0,0 +1,55 @@
+from django.forms import ModelForm, HiddenInput, IntegerField
+from familias.models import Integrante, Tutor
+from .models import Transaccion, Ingreso
+
+
+class TransaccionForm(ModelForm):
+    """ Model form for Transaccion
+
+    This is the general form for updating and creating a
+    transaccion.
+    """
+
+    id_transaccion = IntegerField(required=False, widget=HiddenInput())
+
+    class Meta:
+        model = Transaccion
+        fields = ('monto',
+                  'periodicidad',
+                  'observacion',
+                  'es_ingreso',
+                  'familia')
+        widgets = {
+            'es_ingreso': HiddenInput(),
+            'familia': HiddenInput()
+        }
+
+    def __init__(self, *args, **kwargs):
+        # Add the class form-control to all of the fields
+        super(TransaccionForm, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
+            field.widget.attrs['class'] = 'form-control'
+
+
+class IngresoForm(ModelForm):
+    """ Model form for Ingreso
+
+    This is the general form for creating and updating an
+    ingreso.
+    """
+
+    class Meta:
+        model = Ingreso
+        fields = ('fecha',
```

Comment:
this should have some sort of explanation in the label. It's not obvious what is it referring to